srv/modules/runners/cephservices.py

```diff
@@ -108,22 +109,34 @@ def wait(**kwargs):
     checked.  Raising an exception is ugly but does stop the process.
     """
     settings = {
-        'timeout': 900,
+        'timeout': _timeout(**kwargs),
         'delay': 3
     }
     settings.update(kwargs)
 
     end_time = time.time() + settings['timeout']
     current_delay = settings['delay']
     while end_time > time.time():
-        if check():
+        if check(**kwargs):
             log.debug("Services are up")
             return True
-        time.sleep(settings['delay'])
+        time.sleep(current_delay)
         if current_delay < 60:
-            current_delay += delay
+            current_delay += settings['delay']
         else:
             current_delay = 60
     log.error("Timeout expired")
     raise RuntimeError("Timeout expired")
 
+def _timeout(cluster='ceph', **kwargs):
```

Comment:
What is the motivation for passing ```**kwargs``` here? Its never accessed. I can see why the ```check()``` gets it (though is doesn't use it either).